v2/emailpassword/common-customizations/signup-form/adding-fields.mdx

```diff
@@ -2,26 +2,216 @@
+
+<NpmOrScriptTabs>
+<TabItem value="npm">
+
+```tsx
+import { ^{webjsEmailPasswordSignUp} } from "supertokens-web-js/recipe/^{codeImportRecipeName}";
+
+async function signUpClicked(formData) {
+    try {
+        let response = await ^{webjsEmailPasswordSignUp}({
+            formFields: [{
+                id: "email",
+                value: formData.email
+            }, {
+                id: "password",
+                value: formData.password
+            },{
+                id: "name",
+                value: formData.name
+            }, {
+                id: "age",
+                value: formData.age
+            }, {
+                id: "country",
+                value: formData.country
+            }]
+        })
+
+        if (response.status === "FIELD_ERROR") {
+            // one of the input formFields failed validaiton
+            response.formFields.forEach(formField => {
+                if (formField.id === "email") {
+                    // Email validation failed (for example incorrect email syntax),
+                    // or the email is not unique.
+                    window.alert(formField.error)
+                } else if (formField.id === "password") {
+                    // Password validation failed.
+                    // Maybe it didn't match the password strength
+                    window.alert(formField.error)
+                }
+            })
+        } else if (response.status === "SIGN_UP_NOT_ALLOWED") { 
+            // this can happen during automatic account linking. Tell the user to use another
+            // login method, or go through the password reset flow.
+        } else {
+            // sign up successful. The session tokens are automatically handled by
+            // the frontend SDK.
+            window.location.href = "/homepage"
+        }
+    } catch (err: any) {
+        if (err.isSuperTokensGeneralError === true) {
+            // this may be a custom error message sent from the API by you.
+            window.alert(err.message);
+        } else {
+            window.alert("Oops! Something went wrong.");
+        }
+    }
+}
+```
+
+</TabItem>
+<TabItem value="script">
+
+```tsx
+import supertokens^{recipeNameCapitalLetters} from "supertokens-web-js-script/recipe/^{codeImportRecipeName}";
+async function signUpClicked(formData) {
+    try {
+        let response = await supertokens^{recipeNameCapitalLetters}.^{webjsEmailPasswordSignUp}({
+            formFields: [{
+                id: "email",
+                value: formData.email
+            }, {
+                id: "password",
+                value: formData.password
+            }, {
+                id: "name",
+                value: formData.name
+            }, {
+                id: "age",
+                value: formData.age
+            }, {
+                id: "country",
+                value: formData.country
+            }]
+        })
```

Comment:
just this is enough.. don't need to show how to handle the response as thats already done in the setup section.